db/migrate/20161206130507_create_dispatchers.rb

```diff
@@ -0,0 +1,12 @@
+class CreateDispatchers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :dispatchers do |t|
+      t.string :email, :null => false
```

Comment:
[Use the new Ruby 1.9 hash syntax.](https://github.com/bbatsov/ruby-style-guide#hash-literals)